docs/queries/filters.md

```diff
@@ -125,29 +125,42 @@ because the tasks starts before tomorrow. Only one of the dates needs to match.
 
 ### Recurrence
 
--   `is recurring`
--   `is not recurring`
+- `is recurring`
+- `is not recurring`
 
 ### File Path
 
--   `path (includes|does not include) <path>`
-    -   Matches case-insensitive (disregards capitalization).
+- `path (includes|does not include) <path>`
+  - Matches case-insensitive (disregards capitalization).
 
 ### Description
 
--   `description (includes|does not include) <string>`
-    -   Matches case-insensitive (disregards capitalization).
-    -   Disregards the global filter when matching.
+- `description (includes|does not include) <string>`
+  - Matches case-insensitive (disregards capitalization).
+  - Disregards the global filter when matching.
 
 ### Heading
 
--   `heading (includes|does not include) <string>`
-    -   Whether or not the heading preceding the task includes the given string.
-    -   Always tries to match the closest heading above the task, regardless of heading level.
-    -   `does not include` will match a task that does not have a preceding heading in its file.
-    -   Matches case-insensitive (disregards capitalization).
+- `heading (includes|does not include) <string>`
+  - Whether or not the heading preceding the task includes the given string.
+  - Always tries to match the closest heading above the task, regardless of heading level.
+  - `does not include` will match a task that does not have a preceding heading in its file.
+  - Matches case-insensitive (disregards capitalization).
 
 ### Sub-Items
 
--   `exclude sub-items`
-    -   When this is set, the result list will only include tasks that are not indented in their file. It will only show tasks that are top level list items in their list.
+- `exclude sub-items`
+  - When this is set, the result list will only include tasks that are not indented in their file. It will only show tasks that are top level list items in their list.
+
+### Tags
+
+- `tags (include|do not include) <tag>`
+  - Matches case-insensitive (disregards capitalization).
+  - Disregards the global filter when matching.
+  - The `#` is optional.
+  - The match is partial so `tags include foo` will match `#foo/bar` and `#foo-bar`.
+
+#### Tag Query Examples
+
+- `tag includes #todo`
+- `tag does not include #todo`
```

Comment:
This is marked as resolved, but in the test merge I did, it still says `tag`, so I wonder if any edits got lost as part of reverting markdown changes?

Also, at some point it would be worth noting the alternative spellings of `include` and `does not include`...